tools/clang/unittests/HLSL/AllocatorTest.cpp

```diff
@@ -318,7 +320,8 @@ struct Scenario {
     }
     //   one = two = random
     if (maxIdx > 2) {
-      unsigned one = std::min(1 + (randGen() % (maxIdx - 2)), maxIdx);
+      unsigned randIdx = 1 + (randGen() % (maxIdx - 2));
```

Comment:
why is this change (and the next 3 changes below) needed?

Reply:
The linux version of std::min takes its arguments by reference. You can't pass the result of an expression by reference.